src/initialize/initialize.ts

```diff
@@ -15,13 +15,21 @@ import {
     initPaypal,
     setOnAction,
     initPPCPApple,
+    initPpcp,
 } from 'src';
 
 export function initialize(props: IInitializeProps): void{
-    const {alternative_payment_methods} = getOrderInitialData();
+    let {alternative_payment_methods} = getOrderInitialData();
     setOnAction(props.onAction);
 
-    if (alternative_payment_methods){
+    if (alternative_payment_methods) {
+        // Removing ApplePay from the methods if PPCP is one of the methods since PPCP loads ApplePay
+        const loadsPpcp = alternative_payment_methods.some(payment => payment.type === alternatePaymentMethodType.PPCP);
```

Comment:
won't this break non-fastlane ppcp apple pay? I think it uses the same initialize function